mediator/med_fraction_mod.F90

```diff
@@ -721,60 +721,39 @@ subroutine med_fraction_set(gcomp, rc)
        ! -------------------------------------------
        ! Set FBfrac(compatm)
        ! -------------------------------------------
-       if (is_local%wrap%comp_present(compatm)) then
-
-          if (trim(coupling_mode) == 'nems_orig' .or. trim(coupling_mode) == 'nems_frac' ) then
-
-            ! Set maptype according to coupling_mode
-            if (trim(coupling_mode) == 'nems_orig' ) then
-              maptype = mapnstod_consf
-            else
-              maptype = mapconsf
-            end if
 
-            call FB_FieldRegrid(&
-                 is_local%wrap%FBfrac(compice), 'ifrac', &
-                 is_local%wrap%FBfrac(compatm), 'ifrac', &
-                 is_local%wrap%RH(compice,compatm,:),maptype, rc=rc)
-            if (ChkErr(rc,__LINE__,u_FILE_u)) return
-
-            call FB_FieldRegrid(&
-                 is_local%wrap%FBfrac(compice), 'ofrac', &
-                 is_local%wrap%FBfrac(compatm), 'ofrac', &
-                 is_local%wrap%RH(compice,compatm,:),maptype, rc=rc)
-            if (ChkErr(rc,__LINE__,u_FILE_u)) return
+       if (is_local%wrap%comp_present(compatm)) then
 
+          if (trim(coupling_mode) == 'nems_orig' ) then
+             maptype = mapnstod_consf
           else
-
              if (med_map_RH_is_created(is_local%wrap%RH(compice,compatm,:),mapfcopy, rc=rc)) then
-                ! TODO (mvertens, 2019-11-20): this is not being used when ice and atm are on the same grid
-                ! - this needs to be implemented
                 maptype = mapfcopy
```

Comment:
I am curious, do we need this?

Reply:
I suspect this has to do with one of the CESM non-active configurations,  when atm and ice are the same grid. I don't think this is something we will need.